tests/test-distributed-war/src/main/docker/Dockerfile

```diff
@@ -17,5 +17,5 @@ ADD ${project.build.finalName}.war $APP_DESTINATION
 COPY credential.json /credentials/key.json
 ENV GOOGLE_APPLICATION_CREDENTIALS /credentials/key.json
 
-ENV TOMCAT_MODULES_ENABLE distributed-sessions
-ENV TOMCAT_LOGGING_PROPERTIES com.google.cloud.runtimes.tomcat.session.level=ALL
\ No newline at end of file
+ENV TOMCAT_MODULES_ENABLE distributed-sessions,stackdriver-trace
+ENV TOMCAT_LOGGING_PROPERTIES com.google.cloud.runtimes.tomcat.session.level=ALL,gcp.distributed-sessions.enableTrace=true
```

Comment:
there is no integration test for `gcp.distributed-sessions.separateAttributes` - I'd add another instance with that turned on as well.